### PR TITLE
Fix binary cross entropy with logits op bug

### PR DESCRIPTION
**Pull request by BBuf**
fix https://github.com/Oneflow-Inc/OneTeam/issues/1602#issuecomment-1200340755

### CODE REVIEW

File: oneflow/core/functional/impl/nn_functor.cpp [hunk 1]
```diff
@@ -1128,20 +1128,28 @@ class BinaryCrossEntropyWithLogitsLossFunctor : public LossFunctorBase {
                            const std::string& reduction) const {
     MutableAttrMap attrs;
     JUST(attrs.SetAttr<bool>("has_pos_weight", pos_weight.has_value()));
+    std::shared_ptr<Tensor> broadcast_pos_weight;
```
Comment (Flowingsun007) [inline oneflow/core/functional/impl/nn_functor.cpp:1131]: 加点NOTE说明一下吧

**Review by Flowingsun007 — APPROVED** [review]

**Review by mosout — APPROVED** [review]
